apps/sports-helsinki/src/domain/event/eventQueryString.util.ts

```diff
@@ -14,13 +14,12 @@ export type ReturnParams = {
 export const extractLatestReturnPath = (
   queryString: string,
   locale: string,
-  forwardPath: SearchForwardPath = searchForwardPaths.search as SearchForwardPath
+  forwardPath: typeof SEARCH_ROUTES[keyof typeof SEARCH_ROUTES] = SEARCH_ROUTES.SEARCH // TODO: Allow only SEARCH_ROUTE values
```

Comment:
should we make a separate type so we don't have to copy that always?

Reply:
yep, would be a good idea. But we also need to get it to work like designed 😄 . I add the type, but leave the todo-comments.